Cilsil/Log.cs

```diff
@@ -93,6 +116,17 @@ public static void PrintCoverageStats(IEnumerable<MethodDefinition> methods)
         /// </summary>
         public static void WriteLine(string s) => Console.WriteLine(s);
 
+        /// <summary>
+        /// TODO: use https://nlog-project.org or log4net instead of this class.
+        /// </summary>
+        public static void DebugWriteLine(string s)
```

Comment:
I think having just WriteError(), WriteWarning() and WriteInfo() would be sufficient, with the debug flag works on WriteWarning() and WriteInfo(). Don't specify in the name of the method for whether it is debug or not.

Reply:
Just to clarify, you are suggesting that instead of separate methods, add a `bool debug` (default to false) to the current WriteError, WriteWarning, WriteLine methods and then have each call of WriteLine, WriteError, WriteWarning decide whether to write it as debug or not?  e.g. Log.WriteWarning("Unable to parse instruction", true) would indicate that it logs it in debug mode?

Reply:
Updated with this change